cli/test/test_run_parser.py

```diff
@@ -48,6 +49,50 @@ def test_run_parser_local(test_input_dir) -> None:
         for output_file in Path(output_dir).glob("*.json"):
             assert ParserOutput.parse_file(output_file)
 
+            if "html" in str(output_file):
+                assert ParserOutput.parse_file(output_file).html_data.text_blocks != []
+
+            if "pdf" in str(output_file):
+                pdf_data = ParserOutput.parse_file(output_file).pdf_data
+                assert pdf_data.text_blocks != []
+                assert pdf_data.md5sum is not ""
+                assert pdf_data.page_metadata is not []
+
+
+@pytest.mark.filterwarnings("ignore::urllib3.exceptions.InsecureRequestWarning")
+def test_run_parser_local_series(test_input_dir) -> None:
+    """Test that the parsing CLI runs and outputs a file."""
+    with tempfile.TemporaryDirectory() as output_dir:
+        runner = CliRunner()
+
+        result = runner.invoke(
+            cli_main, [str(test_input_dir), output_dir]
+        )
+
+        assert result.exit_code == 0
+
+        # Default config is to translate to English, and the HTML doc is already in
+        # English - so we just expect a translation of the PDF
+        assert set(Path(output_dir).glob("*.json")) == {
+            Path(output_dir) / "test_html.json",
+            Path(output_dir) / "test_pdf.json",
+            Path(output_dir) / "test_no_content_type.json",
+            Path(output_dir) / "test_pdf_translated_en.json",
+        }
+
+        for output_file in Path(output_dir).glob("*.json"):
+            assert ParserOutput.parse_file(output_file)
+
+            if "html" in str(output_file):
+                assert ParserOutput.parse_file(
+                    output_file).html_data.text_blocks != []
+
+            if "pdf" in str(output_file):
+                pdf_data = ParserOutput.parse_file(output_file).pdf_data
+                assert pdf_data.text_blocks != []
```

Comment:
this would be `True` if `pdf_data.text_blocks is None` (idk whether this is possible in the ParserOutput model). it might be worth doing `assert pdf_data.text_blocks` instead to assert any truthy value?